test/runtime/Dockerfile

```diff
@@ -22,7 +22,7 @@ WORKDIR /app
 COPY --from=builder /runtime ./runtime
 
 COPY test/config/docker/config.yml .
-COPY test/config/64d4b984-14c5-4135-88d0-f3ef0b7b82ac.yaml ./assessments/
+COPY test/config/assessments/64d4b984-14c5-4135-88d0-f3ef0b7b82ac.yaml ./assessments/
```

Comment:
@gusfcarvalho

Reply:
I really don't know what that file was doing. What I can do Is just copy the whole `test/config/assessments` onto `./assessments/`,  as to prevent this path failure anyways. That works?